packages/config-plugins/src/plugins/withIosBaseMods.ts

```diff
@@ -218,9 +221,13 @@ const defaultProviders = {
     async read(filePath, config) {
       let modResults: JSONObject;
       try {
-        const contents = await readFile(filePath, 'utf8');
-        assert(contents, 'Entitlements plist is empty');
-        modResults = plist.parse(contents);
+        if (fs.existsSync(filePath)) {
```

Comment:
You know my opinion, I'm against using sync functions. This is inside an async one so you can refactor it easily.